enrich/geo/computeGeo.js

```diff
@@ -0,0 +1,29 @@
+const get = require("lodash/get")
+
+const AirtableGeoField = {
+  COUNTRY: "Country",
+  STATE: "State",
```

Comment:
I wonder if we want to adopt a more global naming convention here. Geonames and Google both call this an "Administrative Region" since state isn't a universal concept globally.

Reply:
"Administrative Region" can work :) 
I do not know anything about conventions for this so lets put what you think is the best

Reply:
Will respond to this in the main comments